lib/entice/logic/movement.ex

```diff
@@ -28,6 +29,9 @@ defmodule Entice.Logic.Movement do
         |> Map.put(Position, new_pos)
         |> Map.put(Movement, new_movement)
       end)
+    {:ok, %MapInstance{map: map, players: _}} = Entity.fetch_attribute(entity, MapInstance)    
```

Comment:
I added it to be able to notify the player's map's agents but I'll remove it now that we're gonna use :entity_change instead.